FBSnapshotTestCase/Categories/UIImage+Compare.h

```diff
@@ -34,4 +34,8 @@
 
 - (BOOL)fb_compareWithImage:(UIImage *)image tolerance:(CGFloat)tolerance;
 
+- (BOOL)fb_matchesImage:(UIImage *)image;
```

Comment:
We can name it `fb_isEqualToImage:` since we check the the bitmap, I would also appreciate some documentation.